lib/hanami/action/request.rb

```diff
@@ -26,11 +26,12 @@ class Request < ::Rack::Request
 
       # @since 2.0.0
       # @api private
-      def initialize(env:, params:, session_enabled: false)
+      def initialize(env:, params:, default_tld_length:, session_enabled: false)
```

Comment:
```suggestion
      def initialize(env:, params:, default_tld_length: 1, session_enabled: false)
```

Let's add a default arg here, just so this class makes sense on its own.

I think this will mean you can avoid having to manually pass `default_tld_length: 1` when constructing the requests in a bunch of the tests you changed.